examples/Demo/FluentUI.Demo.Client/Infrastructure/IAppVersionService.cs

```diff
@@ -0,0 +1,10 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+namespace FluentUI.Demo.Client;
+
+public interface IAppVersionService
```

Comment:
I don't think this interface is useful (even if we want to inject it into Services later). The class is directly usable.